src/configuration/OptionParser.cpp

```diff
@@ -76,6 +76,9 @@ void OptionParser::initialize()
           boost::program_options::value<int>( &((*_intOptions)[Options::VERBOSITY]) ),
           "Verbosity of engine::solve(). 0: does not print anything (for DnC), 1: print"
           "out statistics in the beginning and end, 2: print out statistics during solving." )
+        ( "split-threshold",
+          boost::program_options::value<int>( &((*_intOptions)[Options::SPLIT_THRESHOLD]) ),
+          "Threshold to split" )
```

Comment:
add more explanation

Reply:
perhaps "Max number of tries to fix a relu before splitting"

Reply:
changed comments